tools/rime_api_console.cc

```diff
@@ -187,6 +187,15 @@ void on_message(void* context_object,
   }
 }
 
+bool ensure_session(RimeApi* rime, RimeSessionId& session_id) {
```

Comment:
In Google C++ style, passing by mutable reference is discouraged because output arguments is not clearly marked in the calling site. The suggested alternative is passing by pointer.

In this scenario, since the failing case is zero session id, we can make the function return the new session id directly instead of using an output argument, making the function a wrapper of `RimeApi::create_session` with error logs. The call `(session_id = create_session(rime))` clearer indicates that `session_id` is reassigned.